src/io/flutter/run/daemon/DeviceDaemon.java

```diff
@@ -148,10 +151,17 @@ private static boolean usesFlutter(Project p) {
     private final @NotNull String command;
     private final @NotNull ImmutableList<String> parameters;
 
-    private Command(@NotNull String workDir, @NotNull String command, @NotNull ImmutableList<String> parameters) {
+    /**
+     * The value of ANDROID_HOME to use when launching the command.
+     */
+    private final @Nullable String androidHome;
+
+    private Command(@NotNull String workDir, @NotNull String command, @NotNull ImmutableList<String> parameters,
+                    @Nullable String androidHome) {
```

Comment:
Are there other environment variables we may want to inject as well (e.g., for iOS development)?  If so, maybe we can abstract this a little in the future to be something like an env map.